### PR TITLE
ADEN-10450 Outstream, floor-adhesion and interstitial on mobile-wiki

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

**Review by nandy-andy — APPROVED** [review]

**Review by dmnsobczak — APPROVED** [review]

File: platforms/ucp-mercury/setup/context/slots/ucp-mercury-slots-context.setup.ts [hunk 1]
```diff
@@ -164,7 +164,7 @@ export class UcpMercurySlotsContextSetup implements DiProcess {
 					loc: 'footer',
 					rv: 1,
 				},
-				defaultTemplates: ['floorAdhesion', 'hideOnViewability'],
+				defaultTemplates: ['floorAdhesion'],
```
Comment (Loczi94) [inline platforms/ucp-mercury/setup/context/slots/ucp-mercury-slots-context.setup.ts:167]: Why did you remove `hideOnViewability` from default templates for floor adhesion? 🤔

Comment (fraszczakszymon) [reply platforms/ucp-mercury/setup/context/slots/ucp-mercury-slots-context.setup.ts:167]: It's implemented in new floor-adhesion (templates 2.0)
